xml/System.Buffers/SequenceReaderExtensions.xml

```diff
@@ -14,7 +14,7 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Provides extended functionality for the <see cref="T:System.Buffers.SequenceReader`1" /> class that allows parsing and reading of UTF-8, UTF-16, and binary data.</summary>
```

Comment:
Is this description meant to be future-proofing the type? Because currently it only supports reading numeric values from binary data:
https://github.com/dotnet/corefx/blob/fec76b0fc06b4fce9e4ac28b9a2bf1101209d984/src/System.Memory/ref/System.Memory.cs#L396-L404

Reply:
I think we should only document what we currently support. Does this suggestion sound better?:

```suggestion
    <summary>Provides extended functionality for the <see cref="T:System.Buffers.SequenceReader`1" /> class that allows parsing and reading numeric values from binary data.</summary>
```

Reply:
Remove parsing. There are no parsing capabilities.

Maybe also add "endian-aware"

```suggestion
    <summary>Provides extended functionality for the <see cref="T:System.Buffers.SequenceReader`1" /> class that allows reading of endian-aware numeric values from binary data.</summary>
```